libcudacxx/include/cuda/std/__cmath/abs.h

```diff
@@ -21,9 +21,9 @@
 #  pragma system_header
 #endif // no system header
 
-#include <cuda/std/__cmath/fp_utils.h>
 #include <cuda/std/__cmath/signbit.h>
 #include <cuda/std/__concepts/concept_macros.h>
+#include <cuda/std/__internal/fp.h>
```

Comment:
I would like tot reserve `__internal` for only libcu++ specific code.

This would be better places in an `__floating_point` folder